english/toucheism/touche.md

```diff
@@ -0,0 +1,22 @@
+In life we all crave attention and some put it in one word "love,"
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.